### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by mugberto**
# PROJECT: Re-former

## We created forms using:
- regular html tags
- rails' form_tag helper
- rails' form_for helper

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <%= label_tag :password, 'Password:' %>
+  <%= password_field_tag :password %> <br><br>
+  <%= submit_tag 'Submit'%> <br><br>
+<% end %> -->
+
+<%= form_for @user do |f| %>
+  <%= f.label :username, "Username Here:" %>
+  <%= f.text_field :username %> <br><br>
+  <%= f.label :email, "Email Here:" %>
+  <%= f.text_field :email, placeholder: "email@test.com", value: "email@test.com" %><br><br>
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Kindly remove the value of input field.

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token%>">
+  <input type="submit" value="Submit">
+</form> -->
+
+
+<!-- <%= form_tag('/users', method: :post) do %>
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: It's better to use `users_path` rails helper.

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
```
Comment (maelfosso) [inline app/views/users/new.html.erb:1]: It's better to use `users_path` rails helper.

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: Remove comments

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: Redirect to `edit_user_path`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,
Thank you for applying the suggested changes.

Kindly read the comments left below.
Once done submit a new code review request

Thanks

Mael FOSSO

File: app/views/users/new.html.erb (hunk 3 above)
Comment (Morrism1) [reply app/views/users/new.html.erb:1]: This was a regular HTML form. We used actual action to test it and it is commented out.

**Review by sumancrest0001 — APPROVED** [review]
**Status: APPROVED :100: :1st_place_medal: 
Congratulation TEAM :clap:** 
It is approved as you implemented all the changes requested by the previous TSE. Now, you can move to the next project.
Happy coding !!!
[Optional] you can use email_field_tag for emails instead of just plain text_field_tag.

**Pull request merged**